packages/react-data-grid/style/rdg-checkbox.less

```diff
@@ -1,19 +1,19 @@
-.react-grid-checkbox, .radio-custom {
```

Comment:
I don't know what these "custom" classes are.

Reply:
It does not seem like they were ever used, I also checked master and no references there also